test/decompiler/test_FormExpressionBuild2.cpp

```diff
@@ -1161,10 +1161,10 @@ TEST_F(FormRegressionTest, Method4ResTag) {
   std::string type = "(function res-tag int)";
   std::string expected =
       "(the-as int (if (zero? (-> arg0 inlined?))\n"
-      "            (* (-> arg0 elt-count) 4)\n"
-      "            (* (-> arg0 elt-count) (-> arg0 elt-type size))\n"
+      "                (* (-> arg0 elt-count) 4)\n"
+      "                (* (-> arg0 elt-count) (-> arg0 elt-type size))\n"
```

Comment:
why did this change? This looks like the right thing after the pretty printer change, but why wasn't this failing tests earlier?

Reply:
ugh i don't know -- i think i knew at the time but ive forgotten.  my changes seem functionally equivalent except whitespace, maybe it's no longer a problem